integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -444,6 +450,172 @@ public void testLivenessProbe() {
+
+    // check in admin server pod, there is no data file for JMS server created
+    String dataFileToCheck = DATA_HOME_OVERRIDE + "/" + domainUid + "/FILESTORE-0000000.DAT";
+    String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+    assertFalse(assertDoesNotThrow(
+        () -> doesFileExistInPod(domainNamespace, adminServerPodName, dataFileToCheck),
+        String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace)),
+        String.format("%s exists in pod %s in namespace %s, expects not exist",
+            dataFileToCheck, adminServerPodName, domainNamespace));
+
+    // check in admin server pod, the default admin server data file moved to DATA_HOME_OVERRIDE
+    String defaultAdminDataFile = DATA_HOME_OVERRIDE + "/" + domainUid + "/_WLS_ADMIN-SERVER000000.DAT";
+    assertTrue(assertDoesNotThrow(() ->
+        doesFileExistInPod(domainNamespace, adminServerPodName, defaultAdminDataFile),
+        String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace)),
+        String.format("can not find file %s in pod %s in namespace %s",
+            defaultAdminDataFile, adminServerPodName, domainNamespace));
+
+    // check in managed server pod, the custom data file for JMS and default managed server datafile are created
+    // in DATA_HOME_OVERRIDE
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = domainUid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      String customDataFile = DATA_HOME_OVERRIDE + "/" + domainUid + "/FILESTORE-0@MANAGED-SERVER" + i + "000000.DAT";
+      assertTrue(assertDoesNotThrow(() ->
+              doesFileExistInPod(domainNamespace, managedServerPodName, customDataFile),
+          String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+              customDataFile, managedServerPodName, domainNamespace)),
+          String.format("can not find file %s in pod %s in namespace %s",
+              customDataFile, managedServerPodName, domainNamespace));
+
+      String defaultMSDataFile = DATA_HOME_OVERRIDE + "/" + domainUid + "/_WLS_MANAGED-SERVER" + i + "000000.DAT";
+      assertTrue(assertDoesNotThrow(() ->
+              doesFileExistInPod(domainNamespace, managedServerPodName, defaultMSDataFile),
+          String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+              defaultMSDataFile, managedServerPodName, domainNamespace)),
+          String.format("can not find file %s in pod %s in namespace %s",
+              defaultMSDataFile, managedServerPodName, domainNamespace));
+    }
+  }
+
+  @Test
+  @DisplayName("Test dataHome override in a domain with model in image type")
+  public void testDataHomeOverrideMiiDomain() {
+    // in this domain, dataHome is not specified
```

Comment:
Move description to javadoc section

Provide a complete sentence  [ in this domain, dataHome is not specified ]
e.g. Create a Domain Resource with defined dataHome.

Reply:
fixed